### PR TITLE
doc: release-notes-3.7: Add info for bt_foreach_bond and HCI Driver

### PR DESCRIPTION
**Pull request by lylezhu2012**
Add information support for BR key traversal by bt_foreach_bond.
Add information support for NXP IW612 Chipset.

### CODE REVIEW

**Review by kartben — DISMISSED** [review]

**Review by aescolar — DISMISSED** [review]

**Comment by aescolar** [comment]
@lylezhu2012 rebase needed

**Comment by lylezhu2012** [comment]
> @lylezhu2012 rebase needed

It is done.

**Review by aescolar — DISMISSED** [review]

File: doc/releases/release-notes-3.7.rst [hunk 1]
```diff
@@ -358,6 +360,8 @@ Bluetooth
   * Added support for Infineon CYW208XX.
   * Added support for Renesas SmartBond DA1469x.
   * Removed unmaintained B91 driver.
+  * Added support for NXP IW612 Chipset on boards mimxrt1170_evkb and mimxrt1040_evk.
+    It can be enabled by :kconfig:option:`CONFIG_BT_NXP_NW612`.
```
Comment (jhedberg) [inline doc/releases/release-notes-3.7.rst:364]: Is this different from the "Added support for NXP platforms" a few lines earlier? If so, maybe that other line needs specifying exactly what (other) NXP platforms it's referring to?

File: doc/releases/release-notes-3.7.rst [hunk 2]
```diff
@@ -349,6 +349,8 @@ Bluetooth
     :kconfig:option:`CONFIG_BT_PER_ADV_SYNC_TRANSFER_SENDER` now depend on
     :kconfig:option:`CONFIG_BT_CONN` as they do not work without connections.
 
+  * Improve :c:func:`bt_foreach_bond` to support Bluetooth classic key traversal.
```
Comment (jhedberg) [inline doc/releases/release-notes-3.7.rst:352]: Should "classic" be "Classic"?

**Review by jhedberg — CHANGES_REQUESTED** [review]

File: doc/releases/release-notes-3.7.rst (hunk 1 above)
Comment (lylezhu2012) [reply doc/releases/release-notes-3.7.rst:364]: I think they are different. "Added support for NXP platforms" mainly refers to a platform where the host and controller are on the same chip. And what I want to describe is that a separate Bluetooth controller module is enabled in this release.
@dleach02 Is it correct?

Comment (lylezhu2012) [reply doc/releases/release-notes-3.7.rst]: Have udpated.

**Review by jhedberg — APPROVED** [review]

**Review by aescolar — APPROVED** [review]

**Pull request merged**